### PR TITLE
Constraint docker-swarm

### PR DESCRIPTION
**Pull request by yoavrotems**
Add docker-swarm as a constraint and auto-detect if swarm is active.
Fix https://github.com/aquasecurity/docker-bench/issues/61 
But need to wait for https://github.com/aquasecurity/docker-bench/pull/79 and https://github.com/aquasecurity/bench-common/pull/102 to be merged. 
Until then leaving it as a draft

### CODE REVIEW

**Comment by lizrice** [comment]
Does this need a new release of bench-common?

**Comment by yoavrotems** [comment]
Yes we need to cut a new release for bench-common and then it should work just fine

File: app_test.go [hunk 1]
```diff
@@ -45,14 +45,15 @@ func TestGetControls(t *testing.T) {
 		t.Errorf("unexpected error: %s\n", err)
 	}
 
-	_, err = getControls(path, "")
+	_, err = getControls(path, "",nil)
```
Comment (lizrice) [inline app_test.go:48]:
```suggestion
	_, err = getControls(path, "", nil)
```

File: app_test.go [hunk 2]
```diff
@@ -61,7 +62,7 @@ func TestRunControls(t *testing.T) {
 	if err != nil {
 		t.Errorf("unexpected error: %s\n", err)
 	}
-	control, err := getControls(path, configPath)
+	control, err := getControls(path, configPath,nil)
```
Comment (lizrice) [inline app_test.go:65]:
```suggestion
	control, err := getControls(path, configPath, nil)
```

File: app.go [hunk 3]
```diff
@@ -86,7 +92,7 @@ func runControls(controls *check.Controls, checkList string) check.Summary {
 	return summary
 }
 
-func getControls(path string, substitutionFile string) (*check.Controls, error) {
+func getControls(path string, substitutionFile string,constraints []string) (*check.Controls, error) {
```
Comment (lizrice) [inline app.go:95]:
```suggestion
func getControls(path string, substitutionFile string, constraints []string) (*check.Controls, error) {
```

**Review by lizrice — APPROVED** [review]
Some spaces (!) but otherwise LGTM

**Pull request merged**